sdk/inc/azure/iot/az_iot_pnp_client.h

```diff
@@ -674,7 +679,7 @@ AZ_NODISCARD az_result az_iot_pnp_client_property_builder_end_reported_status(
  */
 AZ_NODISCARD az_result az_iot_pnp_client_property_get_property_version(
     az_iot_pnp_client const* client,
-    az_json_reader json_reader,
+    az_json_reader* json_reader,
```

Comment:
Is it possible to make this `az_json_reader const*` since it is an [in]?  I recall having trouble with that at some point because it later was cast to a void*???  Not sure if that is relevant.  Also, I recollect something about not wanting the json structure to be altered, which is way it was originally pass by value.  Would const alleviate this (I haven't looked at the az_json_reader structure)?

Reply:
yes you are right it should be `const`

Reply:
By that do you mean you expect the az_json_reader to be altered?  In that case, would the header description be [in,out] ?
@antkmsft Can you please refresh me on this lol. I don't know if what I'm recalling is accurate.